src/main/java/plugily/projects/villagedefense/arena/managers/ShopManager.java

```diff
@@ -136,7 +136,7 @@ private void registerShop() {
       if(itemStack.hasItemMeta() && itemStack.getItemMeta().hasLore()) {
         for(String s : ComplementAccessor.getComplement().getLore(itemStack.getItemMeta())) {
           if(s.contains(plugin.getChatManager().colorMessage(Messages.SHOP_MESSAGES_CURRENCY_IN_SHOP)) || s.contains("orbs")) {
-            costString = ChatColor.stripColor(s).replaceAll("[^0-9]", "");
+            costString = ChatColor.stripColor(s).replaceAll("&[0-9a-zA-Z]", "").replaceAll("[^0-9]", "");
```

Comment:
`ChatColor#stripColor` already do this.

Reply:
@montlikadani
In Spigot and Paper prior to the latest build, yes.
But your latest commit on Commons_Box changed the serializer to `LegacyComponentSerializer.legacyAmpersand()`.
The result is that the string from `getComplement().getLore()` will return the raw string (`&650 orbs`), and `ChatColor#stripColor` itself will not remove the `&6`

Reply:
You can test yourself by 
- Using the latest Paper 1.16.5 with Adventure
- Compiling with the latest `Commons-Box`
- Checking the `s` in the lore loop and the parsed `cost`